src/examples/fake_probing_demo.rb

```diff
@@ -10,5 +10,7 @@
 sdy = ::Storage::Disk.create(devicegraph, "/dev/sdy")
 sdz = ::Storage::Disk.create(devicegraph, "/dev/sdz")
 fake_probing.to_probed
+
+probed = Yast::Storage::StorageManager.instance.probed
 puts("Probed disks:")
-fake_probing.dump_disks(Yast::Storage::StorageManager.instance.probed)
+probed.all_disks.each { |disk| puts("  Found disk #{disk.name}") }
```

Comment:
This had been a debugging function to begin with. Now we can dump the entire tree to YAML wich is a lot more informative.